chunk/storage_client.go

```diff
@@ -13,11 +18,9 @@ type StorageClient interface {
 	// For the read path
 	QueryPages(ctx context.Context, entry IndexEntry, callback func(result ReadBatch, lastPage bool) (shouldContinue bool)) error
 
-	// For table management
-	ListTables() ([]string, error)
-	CreateTable(name string, readCapacity, writeCapacity int64) error
-	DescribeTable(name string) (readCapacity, writeCapacity int64, status string, err error)
-	UpdateTable(name string, readCapacity, writeCapacity int64) error
+	// For storing and retrieving chunks
```

Comment:
Trailing period missing in comment (https://golang.org/doc/effective_go.html#commentary)

Ditto in whole struct.